docs/source/config_file.rst

```diff
@@ -197,18 +197,19 @@ section of the command line docs.
 
 .. confval:: exclude
 
-    :type: newline separated list of regular expressions
+    :type: regular expressions
```

Comment:
It sounds like a single regular expression?

```suggestion
    :type: regular expression
```

Reply:
Good spotting!